tests/test_app/settings.py

```diff
@@ -78,9 +78,17 @@
 LANGUAGE_CODE = "en-gb"
 TIME_ZONE = "Europe/London"
 USE_I18N = True
+LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
 USE_L10N = True
 USE_TZ = True
 
+# English default
+gettext = lambda s: s  # NOQA
+LANGUAGES = (
+    ("en", gettext("English")),
+    ("de", gettext("German")),
```

Comment:
I'll add Spanish 😄

Reply:
You can help with this if you want, i need to whittle down all the strings in this translation file until its just jazzmin ones, there should be about 5 strings left